roles/rhoai/README.md

```diff
@@ -28,7 +42,7 @@ The default operator map looks something like this:
 servicemesh:
   package: servicemeshoperator
   channel: stable
-namespace: openshift-servicemesh
+namespace: openshift-operators
```

Comment:
you only updated the doc but didn't update `rhoai_default_operator_map` in `vars/main.yml`